Gemfile

```diff
@@ -3,15 +3,16 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "2.7.1"
 
+gem "dotenv-rails", groups: [:development, :test]
```

Comment:
please move this gem down to group :development, :test

Reply:
done sir